src/ensembl/src/shared/inline-svg/InlineSvg.tsx

```diff
@@ -0,0 +1,46 @@
+/*
+The purpose of this component is to get an svg image over the network,
+and then to inline it into the DOM (instead of assigning it as a src attribute
+to the img element).
+
+The reason for doing so is that inline svg’s can be styled with CSS,
+whereas the svg’s added using the img element cannot
+*/
+
+import React, { useState, useEffect, useRef } from 'react';
+
+import apiService from 'src/services/api-service';
+
+import styles from './InlineSvg.scss';
+
+type Props = {
+  src: string; // url of the svg
+};
+
+const InlineSVG = (props: Props) => {
+  const [svg, setSvg] = useState<string | null>(null);
+  const [isSet, setIsSet] = useState(false);
```

Comment:
`setIsSet` sounds a bit weird, but that's probably due to `isSet`. How about `isSvgSet` and `setIsSvgSet`?

Reply:
Renamed the variables.